typings/internal-ambient.d.ts

```diff
@@ -29,7 +29,8 @@ declare namespace NodeJS {
     setHiddenValue<T>(obj: any, key: string, value: T): void;
     deleteHiddenValue(obj: any, key: string): void;
     requestGarbageCollectionForTesting(): void;
-    createDoubleIDWeakMap(): any;
+    createIDWeakMap(): ElectronInternal.KeyWeakMap<number, object>;
```

Comment:
why `object` here? Perhaps this could be:
```suggestion
    createIDWeakMap<V>(): ElectronInternal.KeyWeakMap<number, V>;
```